src/credentials/providers/credentialsProvider.ts

```diff
@@ -11,4 +11,5 @@ export interface CredentialsProvider {
     getHashCode(): string
     getCredentials(): Promise<AWS.Credentials>
     canAutoConnect(): boolean
+    isSsoProfile(): boolean
```

Comment:
Is this needed? Doe `getCredentialsProviderId().credentialType` not already indicate "sso" somehow?

Reply:
Currently it returns 'profile', which is the only credentialType supported right now and on all SharedCredential profile.  An update to this PR uses a different solution and this property has been removed.